jenkins/image_building/dib_elements/centos-ci/post-install.d/99-prepull-images

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Prepull container images that are later used in CI tests
+set -uex
+
+export CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-"podman"}
+# Container images that we pre-pull into the CI image.
+export IMG_REGISTRY="${IMG_REGISTRY:-"docker.io/registry:latest"}"
+export IMG_GOLANG_IMG="${IMG_GOLANG_IMG:-"docker.io/golang:1.22"}"
```

Comment:
1.22 ? have we bumped go to 1.22 already?

Reply:
No, we are not yet using 1.22. 
Changed to 1.21